scripts/agegroup.csv

```diff
@@ -0,0 +1,10 @@
+Agegroup_asheagegroup,agegroup

```

Comment:
If we want to commit the source files it would be good to keep them organized - possibly in /scripts/source/?